dffml/util/cli/cmd.py

```diff
@@ -126,6 +128,12 @@ def add_subs(self, add_from: "CMD"):
                 self.add_argument(name.replace("_", "-"), **positional_arg)
             position_list.clear()
 
+        # Add `-log` argument if it's not already added
+        try:
+            self.add_argument(log_cmd.name, **log_cmd)
```

Comment:
Why the `try` block here?

Reply:
That's for cases when `-log` is already an argument for that command, it throws an `ArgumentError` for conflicting option string in those cases.

Reply:
Ah okay great, let's `except ArgumentError` then :)